src/vunnel/providers/amazon/parser.py

```diff
@@ -92,13 +92,13 @@ def _get_alas_html(self, alas_url, alas_file, skip_if_exists=True):
                 content = fp.read()
             return content  # noqa: RET504
         try:
-            r = http.get(alas_url, self.logger, timeout=self.download_timeout)
+            r = http.get(alas_url, self.logger, retries=0, timeout=self.download_timeout)
```

Comment:
I think `0` is too few retries.

This also doesn't handle 403s specifically, which I thought was the intent of the PR. To handle specific status codes differently from other, you can pass the argument `status_handler` to http.get.